test/surface_test.py

```diff
@@ -3848,8 +3848,8 @@ def test_overlap_check(self):
     # https://github.com/pygame/pygame/issues/370#issuecomment-364625291
     @unittest.skipIf("ppc64le" in platform.uname(), "known ppc64le issue")
     def test_colorkey(self):
-        # Check a workaround for an SDL 1.2.13 surface self-blit problem
```

Comment:
I like the retargeted issue link, but I feel we should keep the specificity around the problem offered by these exact details.

Reply:
I would generally agree, but this specificity is IMHO useless if the version in question is 15 years old ;) If it's a SDL 2.x.x version, it definitely makes sense to have a mention of it

Reply:
I've had to go deep into things with SDL_ttf, I appreciated that the pygame comments mentioned exact versions, even if they are presently obsolete. Having the extra bit of information in the comments does no harm.

Reply:
I've reverted that change